test/fixtures/service_fixtures.js

```diff
@@ -304,6 +304,48 @@ module.exports = {
     }
   },
 
+  validUpdateMerchantNameRequest: (value) => {
+    const data = {
+      op: 'replace',
+      path: 'merchant_details/name',
+      value: value
+    }
+
+    return {
+      getPactified: () => {
+        return pactServices.pactify(data)
+      },
+      getPlain: () => {
+        return _.clone(data)
+      }
+    }
+  },
+
+  validUpdateMerchantNameResponse: opts => {
+    opts = opts || {}
+    const externalId = opts.external_id || 'externalId'
+    const serviceName = opts.name || 'updated-service-name'
+    const merchantDetails = opts.merchant_details || {}
+    const merchantName = merchantDetails.name || 'updated-merchant-details-name'
+
+    const data = {
+      external_id: externalId,
+      name: serviceName,
+      merchant_details: {
+        name: merchantName
+      }
+    }
+
+    return {
+      getPactified: () => {
+        return pactServices.pactify(data)
+      },
+      getPlain: () => {
+        return _.clone(data)
+      }
+    }
+  },
```

Comment:
This fixture isn't needed, we should just use `validServiceResponse` as the request returns a service.

Reply:
nuked